README.md

```diff
@@ -3147,6 +3147,299 @@ $ curl -s "https://mainnet.aeternity.io/mdw/v2/transfers?account=ak_7myFYvagcqh8
 
 ---
 
+## NFTs (AEX-141 contracts and tokens)
```

Comment:
will you also add a reference in the directory?

Reply:
@thepiwo So far the NFT images are not downloaded or are you imagining other directory use?

Reply:
I mean in the headlines of the readme to add a reference to this section

Reply:
Done